### PR TITLE
Healer Balance Changes

### PR DESCRIPTION
**Pull request by Ultimate-Fluff**
# About the pull request
This pull request addresses the boring and low-risk gameplay of the Healer drone, who spends half the round sitting next to recovery nodes and recovering her health so she may use it again, rinse and repeat until a rine notices said drone has purple on it and booms her.

First, by changing her health from 600 to 500, Healer can spend more time healing her sisters than sitting through another 100 health to heal herself. Though this makes her less tanky than before, healing classes are not known to be tanks. To ensure Healer can still heal five times without depleting too much of her health whilst still giving her sisters a decent amount of heals, I made her ability cost 75 health instead of 100, and also made her ability cost 200 plasma. Since Healer replenishes plasma much more quickly than her health, she can still put herself into crit if she heals too frequently.  Due to this buff, her heals had a slight nerf, being 10 damage a second for ten seconds instead of 12 damage per second for ten seconds for a total of 20 less damage healed per application overall. 

In addition to these changes, I'm giving Healer a better plasma transfer for when she has nobody else to heal/nowhere else to weed and she has an opportunity to assist her sisters. While a normal drone transfers 50 plasma with a delay of 20, Healer transfers 100 with a delay of 15, which is nowhere near Hivelord's gargantuan 200 plasma with a delay of 5, but it still is better than a normal drone.

Finally, to give the huggers and larva some love, Healer will specifically heal little ones 1.5 health per second for 10 seconds for 15 of her own health and 30 plasma.

# Explain why it's good for the game
Healer drone isn't fun. You run around and heal a bunch of T3s, then sit out for half the battle trying to heal that massive 600 heath while you wonder why you take so long to heal even though you have Strong pheromones. You cry to mom for help, but she doesn't have time to heal a drone who can't build walls and has no need to weed at the moment. You think, 'screw it, I'm going to make a recovery node and camp here until I heal', but by the time you finish healing, several T3s and a silly rouny just suicided into a wall of talls and destroyed your recovery node, so you run off and make another one. But oh, someone noticed you have purple on your carapace and decide your location is precisely where a shell should land, right as you're building one.

No more. These changes allow Healer to move around at her leisure and makes Healer more engaging by allowing her to be a more front-line participant and actively run around and heal her sisters without having to incur such a harsh penalty.

Let this be a testmerge, please.

# Changelog

:cl: Puckaboo2
balance: Healer Drone's health was reduced to 500 from 600.
balance: Healer's damage has been increased to 17 from 12 and the tackle damage debuff has been halved.
balance: Healer Drone's Apply Salve ability now costs 75 health and 200 plasma, down from 120 health and up from 0 plasma.
balance: Healer Drone's Apply Salve ability now heals 10 damage per second for 10 seconds, down from 12 damage per second for ten seconds.
balance: To prevent spam healing between Healers, Apply Salve costs 100 health instead of 75 health when Healer heals another Healer. Much healing.
balance: Healer has an improved Transfer Plasma that gives 100 plasma instead of 50, with a 25% shorter delay. 
balance: Healer will heal huggers and larva for 1.5 health a second for 10 seconds, costing 15 health and 30 plasma. 
tweak: Healer will now face the xeno she is healing if she was not facing their direction before.
spellcheck: All instances of VERYSMALL and VERYLARGE have been renamed to VERY_SMALL and VERY_LARGE.
/:cl:

### CODE REVIEW

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm [hunk 1]
```diff
@@ -106,16 +106,19 @@
 		return
 
 	if(health <= 0)
-		to_chat(src, SPAN_WARNING("You do not have enough health to make regenerative resin!"))
+		to_chat(src, SPAN_WARNING("You do not have enough health to make a regenerative resin salve!"))
+		return
+
+	if(plasma_stored > 200)
```
[the remark this answers is not in the document]
Comment (Ultimate-Fluff) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm:112]: Woop sorry

**Comment by Ultimate-Fluff** [comment]
> Once you figure that out I'm down to TM this. I think the current healer position is rather lame and making them more active would make it a more fun strain.

I have figured it out. I removed the health warning because  of course abilities are disabled whilst a xeno is unconscious. However, I did add a warning Healer is too low to plasma and another warning for when a xeno has 100 or less health from full and doesn't need heals.

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm [hunk 2]
```diff
@@ -42,6 +41,16 @@
...
+/*
+	Improved Plasma Transfer
+*/
+
+/datum/action/xeno_action/activable/transfer_plasma/healer //Improved plasma transfer, but not as much as Hivey.
+	ability_primacy = XENO_PRIMARY_ACTION_3
+	plasma_transfer_amount = 100
+	transfer_delay = 15
+	max_range = 1
+
```
[the remark this answers is not in the document]
Comment (Ultimate-Fluff) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm:53]: Thank you. I've fixed both issues!

**Review by morrowwolf — APPROVED** [review]
Architect approval.

I'll need to go over the code again. Remind me Monday if I haven't done it yet.

**Review by morrowwolf — APPROVED** [review]
Code looks fine

**Pull request merged**